frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -284,13 +286,76 @@ const ManageHostsPage = ({
 
   // ========= routeParams
   const { active_label: activeLabel, label_id: labelID } = routeParams;
-  const selectedFilters = useMemo(() => {
+  const selectedLabels = useMemo(() => {
     const filters: string[] = [];
     labelID && filters.push(`${LABEL_SLUG_PREFIX}${labelID}`);
     activeLabel && filters.push(activeLabel);
     return filters;
   }, [activeLabel, labelID]);
 
+  // All possible filter states - these align with ILoadHostsOptions, but state names here can
+  // differ from param names there:
+  //   searchQuery ||
+  //   teamId ||
+  // labelID / active_label
+  //   policyId ||
+  //   macSettingsStatus ||
+  //   policyResponse ||
+  //   softwareId ||
+  //   softwareTitleId ||
+  //   softwareVersionId ||
+  //   softwareStatus ||
+  //   status ||
+  //   osName ||
+  //   osVersionId ||
+  //   osVersion ||
+  //   macSettingsStatus ||
+  //   bootstrapPackageStatus ||
+  //   mdmId ||
+  //   mdmEnrollmentStatus ||
+  //   munkiIssueId ||
+  //   lowDiskSpaceHosts ||
+  //   missingHosts ||
+  //   osSettingsStatus ||
+  //   diskEncryptionStatus ||
+  //   vulnerability
+
+  const runScriptBatchFilterNotSupported = !!(
```

Comment:
I think we technically can support the [same filterds that "transfer" supports](https://github.com/fleetdm/fleet/blob/main/frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx#L1107-L1132).  TBH I don't know how some of these even get applied, but we're using identical code on the back end to parse them, so they should work the same.

Reply:
Interesting...

so [these filters](https://github.com/fleetdm/fleet/blob/28699-expand-batch-script-run-frontend/frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx#L1179-L1196) could be removed from this condition and included [here](https://github.com/fleetdm/fleet/blob/28699-expand-batch-script-run-frontend/frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx#L1853-L1857) then...@rachaelshaw any reason we shouldn't allow these additional filters?